tests/test_integration_coverage_boost.py

```
@@ -572,13 +570,13 @@
                 Exception("Permission denied"),
                 Exception("Container not found"),
             ]
-            
+
             for i, exception in enumerate(failure_scenarios):
```

Ruff (0.12.2)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)